dolphinscheduler-common/src/main/resources/common.properties

```diff
@@ -14,71 +14,50 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
```

Comment:
Can the modification of this document be reverted?

Reply:
How to modify it

Reply:
How to modify it

Reply:
How to modify it

Reply:
![image](https://user-images.githubusercontent.com/4902714/114173866-654c1480-996a-11eb-8ace-027d13c7882c.png)

![image](https://user-images.githubusercontent.com/4902714/114173974-8ad91e00-996a-11eb-80fb-2c94b3c4c1b6.png)

Reply:
As for you, the commit `8b53ed7` introduced the deleted changes, so you can execute `git checkout 8b53ed7~1 -- dolphinscheduler-common/src/main/resources/common.properties`